README.md

```diff
@@ -1,2 +1,26 @@
-# flexy-historical-data-queue
-A generic EBD-based queue for Ewon historical data
+# sc-flexy-historical-data-queue-lib
+
+THE CODE IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND. HMS DOES NOT WARRANT THAT THE FUNCTIONS OF THE CODE WILL MEET YOUR REQUIREMENTS, OR THAT THE OPERATION OF THE CODE WILL BE UNINTERRUPTED OR ERROR-FREE, OR THAT DEFECTS IN IT CAN BE CORRECTED.
+---
+
+## [Table of Contents](#table-of-contents)
+
+1. [Description](#description)
+2. [Developer Documentation](#developer-documentation)
+3. [Dependencies](#dependencies)
+
+---
+
+## [Description](#table-of-contents)
+
+A generic export block descriptor-based queue library for Ewon historical data.
```

Comment:
This should probably explain that its a queue for logged tag values. Just saying general queue makes me think it could be a queue of objects.

Reply:
reworded